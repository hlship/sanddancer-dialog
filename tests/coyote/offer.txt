(setup coyotes offer)
(now) (#chase has completed)
teleport open desert
z
attack coyotes
l
x coyote
turn flashlight on


