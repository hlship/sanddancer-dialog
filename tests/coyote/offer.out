

Middle of Nowhere (in the pickup truck)
The tire tracks from the south stop abruptly here, but where the hell are you?
The desert sand and clumps of pale sagebrush are all your dimming headlights
pick out before barely reaching the concrete building to the north.

Your poor old pickup truck ticks and groans, smashed gracelessly against a tall
Saguaro; wiffs of evaporating gasoline linger in the chill air. On the dash you
can see a piece of jade. The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

> (setup coyotes offer)
Query succeeded: (setup coyotes offer)
> (now) (#chase has completed)
> teleport open desert

Open Desert
The desert stretches around you in all directions, and every inch of sand looks
just like every other.

Nearby: some clumps of sagebrush, a stand of cactus, and a dead cow. Nothing to
write home about.

> z
A moment slips away.

It's over. They've caught you. They surround you, circling, growling, looking
for weakness, moving in for the kill.

> attack coyotes
It's over. The coyotes dive in for the kill, jaws snapping,, mangy shadows
slinking away into the desert. Except for one last coyote who stands his ground.

And then you blink, and realize he's not a coyote after all. Just... just a guy.

He wears a couple of days of stubble, a scuffed and faded denim jacket, and,
ridiculously in the darkness, a pair of cheap sunglasses. He lights a cigarette,
then holds up a hand in annoyance to block your flashlight beam. "Turn that
thing off," he says, and either you do it or it goes off by itself, because the
next thing you know, it's darker and he's closer and the only light comes from
the red glow of the cigarette.

"That's better," he says. He takes a drag on the cigarette, then pulls another
from behind his ear and offers it to you. "Smoke?"

> l
Open Desert
Clouds of dust swirl through the air, and god damn it's dark.

All you can make out in the darkness are some clumps of sagebrush, a stand of
cactus, and a dead cow.

The coyote watches you intently.

> x coyote
You can barely see him in the darkness, can't tell whether he's old or how old,
what color his faded denim jacket is, and with those sunglasses on, you can't
even tell whether he's smirking at you or just twitching his mouth. There's a
weird smell coming off him, sweat and dirt and a little wet dog and something
deeper, earther, desert, rain, all mixed with cigarette smoke.

> turn flashlight on
You think you'd rather keep him on your side.

> 
